motr/m0crate/crate_index.c

```diff
@@ -308,6 +308,8 @@ struct cr_idx_w {
 	struct cr_time_measure_ctx	exec_time_ctx;
 	size_t				exec_time;
 	enum cr_op_selector		op_selector;
+	m0_time_t                       kv_op_acc_time[CRATE_OP_NR];
```

Comment:
how about name exec_time_per_kv_op_type?

Add comment
// see enum cr_opcode, 4 kv op types for now

Reply:
you may want to have `struct cr_idx_w_results` and move measurements outside workload descriptor.

example
struct cr_idx_ops_result {
      const char * c_op_type; // label
      int                 op_type; // comes from cr_opcode::CRATE_OP_PUT
      m0_time_t    total_time_all_ops_s;
      m0_time_t    time_per_op_ns;
}

struct cr_idx_w_results {
     m0_time_t                  total_time; // entire workload
     cr_idx_ops_result      ops_result[CRATE_OP_NR]; // per op result for each put, get, del, next
}

Reply:
Fixed as per discussion.